src/main.py

```diff
@@ -291,20 +287,27 @@ def main():
             label_created = git_handler.ensure_label(label_name, label_desc, label_color)
             
             if not label_created:
-                print(f"Warning: Could not create GitHub label '{label_name}'. PR will be created without a label.")
+                log(f"Could not create GitHub label '{label_name}'. PR will be created without a label.", is_warning=True)
                 label_name = ""  # Clear label_name to avoid using it in PR creation
 
             pr_title = git_handler.generate_pr_title(vuln_title)
 
             updated_pr_body = pr_body_base + qa_section
+            
+            # Create a brief summary for the telemetry aiSummaryReport (limited to 255 chars in DB)
+            # Generate an optimized summary using the dedicated function in telemetry_handler
+            brief_summary = telemetry_handler.create_ai_summary_report(updated_pr_body)
```

Comment:
Should we make this field bigger in the DB? We could make it a `TEXT` field if that would be helpful?